tests/test_onboarding.py

```diff
@@ -0,0 +1,121 @@
+# SPDX-License-Identifier: MIT
+
+from unittest import mock
+
+import pytest
+
+from disnake import (
+    Guild,
+    Object,
+    Onboarding,
+    OnboardingPrompt,
+    OnboardingPromptOption,
+    OnboardingPromptType,
+    PartialEmoji,
+)
+from disnake.types import onboarding as onboarding_types
+
+onboarding_prompt_option_payload: onboarding_types.PartialOnboardingPromptOption = {
+    "title": "test",
+    "description": "test",
+    "emoji": {"id": "123", "name": "", "animated": False},
+    "role_ids": ["456", "789"],
+    "channel_ids": ["123", "456"],
+}
+
+
+@pytest.fixture()
+def onboarding_prompt_option() -> OnboardingPromptOption:
+    return OnboardingPromptOption(
+        title="test",
+        description="test",
+        emoji=PartialEmoji(name="", id=123, animated=False),
+        roles=[Object(id="456"), Object(id="789")],
+        channels=[Object(id="123"), Object(id="456")],
+    )
+
+
+@pytest.fixture()
+def onboarding_prompt() -> OnboardingPrompt:
+    onboarding_prompt_payload: onboarding_types.PartialOnboardingPrompt = {
+        "title": "test",
+        "options": [],
+        "single_select": True,
+        "required": True,
+        "in_onboarding": True,
+        "type": OnboardingPromptType.multiple_choice,  # type: ignore
+    }
+
+    return OnboardingPrompt(**onboarding_prompt_payload)  # type: ignore
```

Comment:
```suggestion
    onboarding_prompt_payload: onboarding_types.OnboardingPrompt = {
        "id": 0,
        "title": "test",
        "options": [],
        "single_select": True,
        "required": True,
        "in_onboarding": True,
        "type": OnboardingPromptType.multiple_choice.value,
    }

    return OnboardingPrompt._from_dict(
        data=onboarding_prompt_payload,
        state=mock.Mock(http=mock.AsyncMock()),
    )
```
This avoids the `type: ignore`s

Reply:
thanks 4b0f2b3d90700a68f681572abd5b99132c4f93f0